### PR TITLE
Updated diffstat to 1.61

### PR DESCRIPTION
**Pull request by scottmc**

### CODE REVIEW

File: dev-util/diffstat/diffstat-1.61.recipe [hunk 1]
```diff
@@ -0,0 +1,49 @@
+SUMMARY="Display a histogram of diff changes"
+DESCRIPTION="Reads the output of diff and displays a histogram of the \
+insertions, deletions, and modifications per-file.  It is useful for \
+reviewing large, complex files."
+HOMEPAGE="http://invisible-island.net/diffstat/diffstat.html"
+SOURCE_URI="ftp://invisible-island.net/diffstat/diffstat-1.61.tgz"
+LICENSE="Diffstat"
+COPYRIGHT="1994-2016  Thomas E. Dickey"
```
Comment (waddlesplash) [inline dev-util/diffstat/diffstat-1.61.recipe]: Double space.

File: dev-util/diffstat/diffstat-1.61.recipe [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+REVISION="1"
+CHECKSUM_SHA256="25359e0c27183f997b36c9202583b5dc2df390c20e22a92606af4bf7856a55ee"
+SOURCE_DIR="$portVersionedName"
```
Comment (waddlesplash) [inline dev-util/diffstat/diffstat-1.61.recipe]: Isn't this the default?

File: dev-util/diffstat/diffstat-1.61.recipe [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	"
+
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc
+	"
+
+BUILD()
+{
+	./configure
```
Comment (waddlesplash) [inline dev-util/diffstat/diffstat-1.61.recipe]: Should be `runConfigure ./configure`.

Comment (scottmc) [inline dev-util/diffstat/diffstat-1.61.recipe]: any idea why haikuporter isn't omitting docDir when I try --omit-dirs $docDir

Comment (waddlesplash) [inline dev-util/diffstat/diffstat-1.61.recipe]: Remove the dollar sign.

**Pull request merged**